test/unit/uk/gov/hmrc/apisubscriptionfields/logging/ApiLoggerSpec.scala

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.uk.gov.hmrc.apisubscriptionfields.logging
+
+import java.io.{ByteArrayOutputStream, PrintStream}
+
+import org.scalamock.scalatest.MockFactory
+import uk.gov.hmrc.apisubscriptionfields.logging.ApiLogger
+import uk.gov.hmrc.play.config.inject.ServicesConfig
+import uk.gov.hmrc.play.test.UnitSpec
+
+import scala.io.Source
+
+class ApiLoggerSpec extends UnitSpec with MockFactory {
```

Comment:
I would not really bother testing a logger.
Why this?
If you use the Play logger, you do not need to add this code :)

Reply:
please see reply above

Reply:
we have complied with your suggestion